torch/_inductor/codegen/cpp_bmm_template.py

```diff
@@ -100,18 +102,22 @@ def __init__(
     @staticmethod
     def get_padded_size(n, block_n, k, should_block_weight):
         if should_block_weight:
-            new_size, padded_n = CppGemmTemplate.get_padded_size(n, block_n, k, should_block_weight)
+            # Tensor is constant or not contiguous, so we will pad and block
+            new_size, padded_n = CppGemmTemplate.get_padded_size(
+                n, block_n, k, should_block_weight
+            )
+            # Add the new batch dimension
             new_size.insert(0, -1)
             return new_size, padded_n
         else:
             new_size = [-1, k, n]
             return new_size, n
 
     @staticmethod
-    def block_weight_irnode(W, new_size, padding):
+    def _block_weight_irnode(W, new_size, padding):
         assert isinstance(W, ir.IRNode)
         if W.get_name() in V.graph.constants:
-            return CppGemmTemplate.block_weight_irnode(W, new_size, padding)
+            return CppGemmTemplate._block_weight_irnode(W, new_size, padding)
```

Comment:
Why making it as an internal method adding underscore prefix?

Reply:
Resolved by combining block_weight to one method. I was able to remove overrides in the BMM subclass by streamlining code.